proto/import_sstpb.proto

```diff
@@ -63,6 +63,27 @@ service ImportSST {
 
     // ClearFiles clear applied file after restore succeed.
     rpc ClearFiles(ClearRequest) returns (ClearResponse) {}
+
+    // Temporarily disable ingest / download / write for data listeners don't support catching import data.
+    rpc DenyImportRPC(DenyImportRPCRequest) returns (DenyImportRPCResponse) {}
```

Comment:
How about `DisableImport` or `PauseImport`?

Reply:
Checking from https://github.com/pingcap/tidb/pull/47001/files, this rpc is more like a switch function.